cwl_wes/config.yaml

```diff
@@ -0,0 +1,165 @@
+          add_operation_fields:
+              x-swagger-router-controller: ga4gh.wes.server  
+          disable_auth: True
+          connexion:
+              strict_validation: True
+              validate_responses: False
+              options:
+                  swagger_ui: True
+                  serve_spec: True
+
+# Logging configuration
+# Cf. https://foca.readthedocs.io/en/latest/modules/foca.models.html#foca.models.config.LogConfig
+log:
+    version: 1
+    disable_existing_loggers: False
+    formatters:
+        standard:
+            class: logging.Formatter
+            style: "{"
+            format: "[{asctime}: {levelname:<8}] {message} [{name}]"
+        long:
+            class: logging.Formatter
+            style: "{"
+            format: "[{asctime}: {levelname:<8}] {message} [{name}]"
+    handlers:
+        console:
+            class: logging.StreamHandler
+            level: 20
+            formatter: standard
+            stream: ext://sys.stderr
+    root:
+        level: 10
+        handlers: [console]
+
+# Background job configuration
+# Cf. https://foca.readthedocs.io/en/latest/modules/foca.models.html#foca.models.config.JobsConfig
+jobs:
+    host: rabbitmq
+    port: 5672
+    backend: 'rpc://'
+    include:
+        - cwl_wes.tasks.tasks.run_workflow
+        - cwl_wes.tasks.tasks.cancel_run
+
+# Exception configuration
+# Cf. https://foca.readthedocs.io/en/latest/modules/foca.models.html#foca.models.config.ExceptionConfig
+exceptions:
+    required_members: [['message'], ['code']]
+    status_member: ['code']
+    exceptions: cwl_wes.exceptions.exceptions
+
+# Custom configuration
+# Available in app context as attributes of `current_app.config.foca`
+custom:
+    storage:
+        permanent_dir: '/data/output'
+        tmp_dir: '/data/tmp'
+        remote_storage_url: 'ftp://ftp-private.ebi.ac.uk/upload/foivos'
+    celery:
+        monitor:
+            timeout: 0.1
+        message_maxsize: 16777216
+    endpoint_params:
+        default_page_size: 5
+        timeout_cancel_run: 60
+        timeout_run_workflow: null
+    service_info:
+        contact_info: 'https://github.com/elixir-cloud-aai/cwl-WES'
+        auth_instructions_url: 'https://www.elixir-europe.org/services/compute/aai'
+        supported_filesystem_protocols:
+            - ftp
+            - https
+            - local
+        supported_wes_versions:
+            - 1.0.0
+        workflow_type_versions:
+            CWL:
+                workflow_type_version:
+                    - v1.0
+        workflow_engine_versions:
+            cwl-tes: 0.2.0
+        default_workflow_engine_parameters:
+            - type: string
+              default_value: some_string
+            - type: int
+              default_value: '5'
+        tags:
+            known_tes_endpoints: 'https://tes.tsi.ebi.ac.uk/|https://tes-dev.tsi.ebi.ac.uk/|https://csc-tesk.c03.k8s-popup.csc.fi/|https://tesk.c01.k8s-popup.csc.fi/'
```

Comment:
Just a detail, but the `c03.k8s-popup.csc.fi` cluster has been retired. Maybe not for this MR, but the examples could be fixed one of these days :)